packages/dmn-js-decision-table/src/features/decision-table-head/editor/index.js

```diff
@@ -17,7 +17,8 @@ export default {
     ContextMenu,
     DebounceInput,
     ExpressionLanguagesModule,
-    TypeRefEditing
+    TypeRefEditing,
```

Comment:
Does this do anything? I think we must depend on a module. Depending on a `string` does not yield any meaningful result.